1_preprocessing_data/README.md

```diff
@@ -2,13 +2,37 @@
 
 In this module, we present our pipeline for preprocessing the NF1 Schwann Cell data.
 
-## Illumination Correction
+## Illumination Correction (IC)
 
 To correct for illumination issues within the NF1 data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
 We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
 
 Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
 
+Visualizing if images need IC can be simple or difficult depending on the dataset. 
+In the NF1 dataset, identifying illumination issues is impossible to capture with the human eye.
+
+To visualize the illumination issues, we use Fiji to increase the maximum limits of the display range to brighten the images using the `B&C` tool (Figure 1).
+
+![NF1_IC_Demonstration.png](example_figs/NF1_IC_Demonstration.png)
+> Figure 1. Use of Fiji to Determine Illumination Issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). The increasing of the upper limits for display starts to show where there is brighter areas in the image that was hidden when the upper limit was lower.
+
+The documentation regarding this tool states:
+
+> ImageJ displays images by linearly mapping pixel values in the display range to display values in the range 0-255. Pixels with a value less than the minimum are displayed as black and those with a value greater than the maximum are displayed as white. `Minimum` and `Maximum` control the lower and upper limits of the display range.
+
+After running PyBaSiC and generating corrected images, Fiji is used to confirm if the illumination issues been corrected (Figure 2).
```

Comment:
```suggestion
After running PyBaSiC and generating corrected images, we use Fiji to confirm that the illumination issues been corrected (Figure 2).
```

Active voice :)

Reply:
One day I will get over the passive voice habit from college! Ty!